objects/startup_app.json

```
@@ -0,0 +1,160 @@
+        }
+      }
+    },
+    "start_type": {
+      "description": "The start type of the startup application.",
+      "requirement": "optional"
+    },
+    "start_type_id": {
+      "description": "The start type ID of the startup application.",
+      "requirement": "required"
+    },
+    "type": {
+      "caption": "Type",
+      "description": "The startup application type.",
+      "type": "string_t"
+    },
+    "type_id": {
+      "caption": "Type ID",
+      "description": "The startup application type identifier.",
+      "enum": {
+        "0": {
+          "caption": "Unknown",
+          "description": "The type is unknown."
+        },
+        "1": {
+          "caption": "Kernel Mode Driver",
+          "description": "Kernel mode driver."
+        },
+        "2": {
+          "caption": "User Mode Driver",
+          "description": "User mode driver."
+        },
+        "3": {
+          "caption": "Service",
+          "description": "A background process typically managed by the operating system, e.g., a service process on Windows or a systemd-managed daemon on Linux."
+        },
+        "4": {
+          "caption": "User Mode Application",
+          "description": "An application that runs in the user space."
+        },
+        "5": {
+          "caption": "Autoload",
+          "description": "The macOS Autoload Application."
+        },
+        "6": {
+          "caption": "System Extension",
+          "description": "System extensions on macOS enables 3rd parties to extend the capabilities of macOS."
+        },
+        "7": {
+          "caption": "Kernel Extension",
+          "description": "Kernel extensions on macOS includes Apple provided pre-installs and 3rd party installs which enables support for specific hardware or software features not natively supported by macOS."
+        },
+        "8": {
+          "caption": "Scheduled Job, Task",
+          "description": "A job or task that runs on a configured schedule."
+        },
+        "99": {
+          "caption": "Other",
+          "description": "The startup application type is not mapped. See the <code>type</code> attribute, which contains data source specific values."
+        }
+      },
+      "type": "integer_t"
+    },
+    "driver": {
+      "description": "The startup application kernel driver resource.",
+      "requirement": "optional"
+    },
+    "job": {
+      "description": "The startup application job resource.",
+      "requirement": "optional"
+    },
+    "process": {
+      "description": "The startup application process resource.",
+      "requirement": "optional"
+    },
+    "win_service": {
+      "caption": "Windows Service",
+      "description": "The startup application windows service resource.",
+      "requirement": "optional"
+    }
+  },
+  "constraints": {
+    "just_one": [
+      "driver",
+      "job",
+      "process",
+      "win_service"
+    ]
+  }
```

```suggestion
  "constraints": {
    "just_one": [
      "driver",
      "job",
      "process"
    ]
  }
```
As mentioned above, `constraints` with `win_service` removed. (It will be in the patch though.)